### PR TITLE
APM-317073 - logging AWS log forwarder stack version 

### PR DESCRIPTION
**Pull request by bmrozinski-dt**
in forwarder Lambda's log

### CODE REVIEW

File: src/index.py [hunk 1]
```diff
@@ -28,6 +28,13 @@ def handler(event, lambda_context):
     dt_token = os.environ.get('DYNATRACE_API_KEY')
     verify_SSL = os.environ.get('VERIFY_SSL', 'false') == 'true'
 
+    try:
+        with open('version.txt') as versionFile:
+            version = versionFile.readline()
+    except:
+        version = "?"
+    log_multiline_message("LOG FORWARDER version=" + version, "handler")
+
```
[the remark this answers is not in the document]
Comment (bmrozinski-dt) [reply src/index.py:37]: If we were using our custom logging module like in Azure forwarder, we could think about logging the version every time. But for now, I'd rather leave it as it is.

[the remark this answers is not in the document]
Comment (mmajcher) [reply src/index.py:37]: I also think this one line is enough.

File: src/index.py [hunk 2]
```diff
@@ -28,6 +28,13 @@ def handler(event, lambda_context):
     dt_token = os.environ.get('DYNATRACE_API_KEY')
     verify_SSL = os.environ.get('VERIFY_SSL', 'false') == 'true'
 
+    try:
+        with open('version.txt') as versionFile:
+            version = versionFile.readline()
+    except:
+        version = "?"
```
Comment (mmajcher) [inline src/index.py:35]: Print the exception info, should not be needed but who knows :)

File: src/index.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (dt-be) [reply src/index.py:37]: Yes yes, that's what I meant, I just wanted to have it in plain text.

**Review by dt-be — DISMISSED** [review]

**Review by dt-be — APPROVED** [review]

File: src/index.py [hunk 3]
```diff
@@ -28,6 +28,15 @@ def handler(event, lambda_context):
     dt_token = os.environ.get('DYNATRACE_API_KEY')
     verify_SSL = os.environ.get('VERIFY_SSL', 'false') == 'true'
 
+    try:
+        with open('version.txt') as versionFile:
+            version = versionFile.readline()
+    except Exception as e:
+        version = "?"
+        log_error_with_stacktrace(e, "Couldn't read stack version", "version-reading-exception")
```
Comment (mmajcher) [inline src/index.py:36]: I think this is not error, this should be INFO/WARN as this is expected during local development etc. 
Just INFO/WARN that cannot read version from file + exception.msg 
Thanks

**Review by mmajcher — DISMISSED** [review]

**Comment by kamiljsokolowski** [comment]
@bmrozinski-dt please pull in the latest version of `master` containing fix #48. This will pass your failing build.

**Review by kamiljsokolowski — DISMISSED** [review]

**Review by mmajcher — APPROVED** [review]

**Pull request merged**